test-data/unit/pythoneval.test

```diff
@@ -1365,7 +1365,8 @@ from typing import Dict, List, Tuple
 x: Dict[str, List[int]]
 reveal_type(x['test'][0])
 [out]
-typing.pyi: error: Class typing.Sequence has abstract attributes "__len__"
-typing.pyi: note: If it is meant to be abstract, add 'abc.ABCMeta' as an explicit metaclass
+typing.pyi:251: error: Class typing.Sequence has abstract attributes "__len__"
+typing.pyi:251: note: If it is meant to be abstract, add 'abc.ABCMeta' as an explicit metaclass
+ast.pyi:31: error: Name 'PyCF_ONLY_AST' already defined (possibly by an import)
```

Comment:
This is an actual bug in typeshed, this is name is indeed already defined by a star import.